src/hiero_sdk_python/executable.py

```
@@ -190,111 +430,135 @@ def _execute(self, client: "Client"):
+
+
+def _is_transaction_receipt_or_record_request(request):
+    if not isinstance(request, query_pb2.Query):
+        return False
+
+    return request.HasField("transactionGetReceipt") or request.HasField(
+        "transactionGetRecord"
+    )
+
+
+def _delay_for_attempt(request_id: str, backoff: float, attempt: int, logger, error):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_delay_for_attempt`

Add return type annotation: `None`

(ANN202)